README.md

```diff
@@ -47,7 +47,7 @@ import { ComponentBase } from 'resub';
 import TodosStore = require('./TodosStore');
 
 interface TodoListState {
-    todos?: string[];
+    todos?: String[];
```

Comment:
I don't think this change is correct. Am I missing something? Why do you think String should be capitalized here?